.travis.yml

```diff
@@ -9,7 +9,6 @@ rvm:
   - 2.6.3
   - 2.5.6
   - 2.4.6
-  - 2.3.8
```

Comment:
This is officially deprecated by Ruby so while it doesn't mean we don't support it I don't want it to be part of the Travis builds